habitat-lab/habitat/config/benchmark/nav/instance_imagenav/instance_imagenav_hm3d_v3.yaml

```diff
@@ -0,0 +1,34 @@
+# @package _global_
```

Comment:
Can all of these new benchmarks we used with the gym wrapper ? I would like to be sure they can be.

Reply:
Sorry I am not sure if I know how to properly do this. Do you have any pointers?

Reply:
Hi @vincentpierre, it seems that there is no existing code that tests this on the main branch. Could you please provide more pointers, thank you! @ykarmesh I think this code can be used with the gym wrapper

Reply:
my point is that it seems that we need to create a new test script that covers all the existing yamls